lib/fika/parser/types.ex

```diff
@@ -89,16 +89,31 @@ defmodule Fika.Parser.Types do
     |> reduce({Enum, :join, []})
     |> label("tuple type")
 
-  type =
+  atom =
+    ignore(string(":"))
+    |> concat(identifier_str)
+    |> reduce({Helper, :to_atom, []})
```

Comment:
@emilsoman This combinator was returning `[":", "value"]` instead of `":value"`. Do you see any way to avoid needing this `to_atom` call?